internal/eventstore/handler/handler_projection.go

```diff
@@ -221,11 +228,6 @@ func (h *ProjectionHandler) schedule(ctx context.Context) {
 			}
 			go h.cancelOnErr(lockCtx, errs, cancelLock)
 		}
-		if succeededOnce {
-			// since we have at least one successful run, we can restrict it to events not older than
-			// twice the requeue time (just to be sure not to miss an event)
-			query = query.CreationDateAfter(time.Now().Add(-2 * h.requeueAfter))
-		}
```

Comment:
This filter makes no sense to me.
The query just gets the instance IDs. The filter just excludes instanced that don't have an event since `-2 * h.requeueAfter`.

Reply:
discarded